honeybadger-java/src/main/java/io/honeybadger/reporter/dto/NoticeDetails.java

```diff
@@ -26,18 +26,19 @@ public class NoticeDetails implements Serializable {
     private final Set<String> tags;
     private final Backtrace backtrace;
     private final Causes causes;
+    private final String fingerprint;
```

Comment:
Can you please add `fingerprint` in the `@JsonPropertyOrder` annotation on Line 19 (just for consistency)?